x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/StickySpanProperties.tsx

```diff
@@ -49,7 +67,9 @@ export function StickySpanProperties({ span, totalDuration }: Props) {
 
   const spanName = span.span.name;
   const spanDuration = span.span.duration.us;
-  const spanTypeLabel = getSpanLabel(getPrimaryType(span.span.type));
+  const { type, subtype, action } = getSpanTypes(span);
+  const spanType = formatType(type);
+  const spanSubtype = formatSubtype(subtype);
```

Comment:
Last thing (I promise): can you move these formatters inside `getSpanTypes` (unless we need the unformatted for anything)

Reply:
I still prefer `spanType` over `type` as the exported name for clarity